README.md

```diff
@@ -37,15 +37,45 @@ npm test
 
 ## Usage
 
-### Add/Update a Verifiable Credential
-
-Please add or update credentials in the `./credenitals` dir. When adding new
-credential, make sure the directory name is camel-cased. The directory should
-contain a logo image (with `.jpg` or `.png` file extensions) and the credential
-in the `credential.json` file. Additionally, you can also add a file for the
-credential context in the `./contexts` dir (for example see -
-`./contexts/alumni/v1.json`) if you want to host the contexts in
-[`@credential-handler/vc-playground-contexts-server`](https://github.com/credential-handler/vc-playground-contexts-server).
+### Add / Update a Verifiable Credential
+
+To add or update credentials in the `./credentials` directory:
+
+1. Within the `./credentials` directory, create a new directory for the
+credential. Make sure to use kebab case for the directory name.
+2. Inside the newly created credential directory, add a `credential.json` file.
+This file should contain all the necessary information for the credential,
+including the issuer, credential subject, and other relevant data, for example
+`./credentials/movie-ticket/credential.json`. The credential MUST be conformant
+to [VC Data Model 1.1](https://www.w3.org/TR/vc-data-model#credentials).
+3. Additionally, place a logo `image` file for the credential in the
+corresponding directory. The image file should be in either `JPG` or `PNG`
+format, for example `./credentials/movie-ticket/image.png`.
+
+### Add / Update a Credential Context
+
+To optionally add or update contexts in the `./contexts` directory:
+
+1. If you wish to include a context for a credential, create a new directory
+for the context in the `./contexts` directory. Make sure to use kebab case for
+the directory name. It should contain a file `v1.json` that should contain the
+JSON-LD context definition for the credential, for example
+`./contexts/movie-ticket/v1.json`. The context MUST be conformant
+to [VC Data Model 1.1](https://www.w3.org/TR/vc-data-model/#contexts).
+2. Please note that once a context is created, it becomes locked and cannot be
+modified. If you want to update or make changes to a context, you must create a
+new context file with a new version within the context directory. For instance,
+if you want to update the context for the "Alumni" credential, create a new file
+named `v2.json` containing the updated JSON-LD context definition within
+`./contexts/alumni`. The previous version,`v1.json`, should be retained in the
+directory without any modifications.
+3. Please make sure that the context URLs for the contexts created in the
+`./contexts` directory follow the format:
+`https://contexts.vcplyaground.org/examples/foo-bar/v1.json`.
```

Comment:
This looks like a typo for the domain name.

Reply:
@ottonomy Thank you! Fixed [e4a459f](https://github.com/credential-handler/vc-examples/pull/15/commits/e4a459f7f051efdff6467dbea547438094b69db5)